hydromt/models/model_api.py

```diff
@@ -188,9 +197,7 @@ def update(self, model_out=None, write=True, opt=None):
 
         # check if model has a region
         if self.region is None:
-            raise ValueError(
-                'Model region not found, setup basemaps using "build" method first.'
-            )
+            raise ValueError("Model region not found, setup model using `build` first.")
 
         # remove setup_basemaps from options and throw warning
         if "setup_basemaps" in opt:
```

Comment:
Shall we replace by setup_region ? Or both setup_basemaps+setup_region?

Reply:
good one